tables_io/io_layer.py

```diff
@@ -274,6 +319,47 @@ def readHdf5ToTables(filepath):
     return tables
 
 
+def readHdf5ToDataframe(filepath):
+    """
+    Reads `pandas.DataFrame` objects from an hdf5 file.
+
+    Parameters
+    ----------
+    filepath: `str`
+        Path to input file
+
+    Returns
+    -------
+    df : `pandas.DataFrame`
+        The dataframe
+    """
+    if not HAS_HDF5:  #pragma: no cover
+        raise ImportError("h5py is not available, can't read hdf5")
+    if not HAS_PANDAS:  #pragma: no cover
+        raise ImportError("pandas is not available, can't read hdf5 to dataframe")
+    return pd.read_hdf(filepath)
+
+
+def readHdf5Group(infile, groupname=None):
+    """ Get the size of data in a particular group in an hdf5 file
```

Comment:
Docstring not currently correct - this returns the file and group not the size.

Reply:
Done